packages/auth/src/grant/routes.ts

```diff
@@ -66,7 +71,8 @@ export function createGrantRoutes({
   }
 }
 
-function validateGrantRequest(
+// exported for testing
+export function validateGrantRequest(
```

Comment:
Does this have checks that aren't covered by the OpenAPI validator middleware?
https://github.com/interledger/rafiki/blob/0642a0fa91f892b1398ff10ae07a28c79955b39a/packages/auth/src/app.ts#L239

Reply:
I don't think so but I'd like @njlie to confirm.

Reply:
I'd hold off on any work there now that Adrian has been working on his auth server implementation.

Reply:
> I don't think so but I'd like @njlie to confirm.

Yeah I'm pretty sure this can be excised now, I wrote this before we had brought OpenAPI into the auth service. This basically is a type guard against the `POST /` request which is definitely covered.

Reply:
I removed the validation function and re-opened #351